src/wp-includes/class-wp-theme-json-resolver.php

```diff
@@ -263,13 +263,13 @@ public static function get_user_data_from_custom_post_type( $theme, $should_crea
 
 		// Special case: '-1' is a results not found.
 		if ( -1 === $post_id && ! $should_create_cpt ) {
```

Comment:
```suggestion
		if ( -1 === $post_id && ! $create_post ) {
```
I missed this in resolving the merge conflict.